src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -25,40 +25,27 @@ export function SidebarNavSign() {
         onClickSignMyself: onBoxClickSignMyself,
         status: boxSignStatus,
         targetingApi: boxSignTargetingApi,
-        isSignRemoveInterstitialEnabled,
     } = useFeatureConfig('boxSign');
 
     return (
-        <>
-            {isSignRemoveInterstitialEnabled ? (
-                <DropdownMenu isResponsive constrainToWindow isRightAligned>
-                    <SidebarNavSignButton
-                        blockedReason={boxSignBlockedReason}
-                        status={boxSignStatus}
-                        targetingApi={boxSignTargetingApi}
-                        data-resin-target={SIDEBAR_NAV_TARGETS.SIGN}
-                    />
-                    <Menu>
-                        <MenuItem data-testid="sign-request-signature-button" onClick={onBoxClickRequestSignature}>
-                            <SignMeOthers32 width={16} height={16} className="bcs-SidebarNavSign-icon" />
-                            <FormattedMessage {...messages.boxSignRequestSignature} />
-                        </MenuItem>
-                        <MenuItem data-testid="sign-sign-myself-button" onClick={onBoxClickSignMyself}>
-                            <SignMe32 width={16} height={16} className="bcs-SidebarNavSign-icon" />
-                            <FormattedMessage {...messages.boxSignSignMyself} />
-                        </MenuItem>
-                    </Menu>
-                </DropdownMenu>
-            ) : (
-                <SidebarNavSignButton
-                    blockedReason={boxSignBlockedReason}
-                    data-resin-target={SIDEBAR_NAV_TARGETS.SIGN}
-                    onClick={onBoxClickRequestSignature}
-                    status={boxSignStatus}
-                    targetingApi={boxSignTargetingApi}
-                />
-            )}
-        </>
+        <DropdownMenu isResponsive constrainToWindow isRightAligned>
+            <SidebarNavSignButton
+                blockedReason={boxSignBlockedReason}
+                status={boxSignStatus}
+                targetingApi={boxSignTargetingApi}
+                data-resin-target={SIDEBAR_NAV_TARGETS.SIGN}
+            />
+            <Menu>
+                <MenuItem data-testid="sign-request-signature-button" onClick={onBoxClickRequestSignature}>
```

Comment:
do we need the `data-testid` for anything anymore? if we are using RTL we should need the testid's anymore

Reply:
removed both of them